src/components/LandingPage/index.tsx

```diff
@@ -91,6 +91,41 @@ const LandingPage = ({ handleSelectDataset, megasets }: LandingPageProps) => (
                         </div>
                     </Col>
                 </Row>
+                <Row className={styles.section}>
+                    <Col className={styles.sectionContent}>
+                        <h2 className={styles.subtitle}>Download cell feature data (via Quilt)</h2>
+                        <Row type="flex" className={styles.section}>
+                            <Descriptions layout="horizontal" colon={false} column={1}>
+                                <Descriptions.Item label="RELEASE DATE">DATASET</Descriptions.Item>
+
+                                {downloadData.map((downloadInfo: DownloadInfo) => {
+                                    return (
+                                        <>
+                                            <Descriptions.Item label={downloadInfo.date}>
```

Comment:
What is the reason to use a fragment here? I also see React complaining about a missing "key" prop in this iterated list - if the fragment is present to prevent an error, I'd prefer instead adding a key to `Descriptions.Item`.